src/ProductConstructionService/ProductConstructionService.Api/Configuration/ApiRedirection.cs

```diff
@@ -0,0 +1,191 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Net;
+using System.Net.Http.Headers;
+using Azure.Core;
+using Maestro.Authentication;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.DotNet.Maestro.Client;
+
+namespace ProductConstructionService.Api.Configuration;
+
+internal static class ApiRedirection
+{
+    private const string ApiRedirectionConfiguration = "ApiRedirect";
```

Comment:
I originally put all of those in the configuration class because it didn't make much sense to have them in the startup class if it was used around the place.

This one I moved because I modulized the whole API redirection in a single file and I think it's much easier to maintain this way. I should do the same for the other parts and move the configuration to the code that uses it.

Reply:
I cleaned up Startup and made all the modules manage their own configuration keys so that it's easy to see what belongs under a module and what config it expects.